### ISSUE TITLE
How can i get Email?

### ISSUE THREAD

**Issue by nicoaureliovillanueva** [tipsi/tipsi-twitter#10]
Hi Sir! Nice Work it helps me alot. But i have a question? how can i get "email, first name, last name, birthdate"? i hope you can help me on this.. Thanks alot More powers :)

**Comment by isnifer** [comment]
@nicoaureliovillanueva Hi! Thank you for the issue. We will discover it.

**Comment by nicoaureliovillanueva** [comment]
Thank you very much sir @isnifer i hope getting email from Twitter is possible in this module.. Thanks alot sir!

**Comment by isnifer** [comment]
@nicoaureliovillanueva first of all, about email read this - https://stackoverflow.com/a/37181724/1671555

**Comment by nicoaureliovillanueva** [comment]
HI Sir @isnifer... any update on this issue? Thank you very much sir :)

**Comment by isnifer** [comment]
@nicoaureliovillanueva As I said, you can't get an email from Twitter.
About other fields read this - https://dev.twitter.com/twitterkit/ios/log-in-with-twitter.

TLDR; 
You need to use Twitter REST API. This library doesn't provide it.

**Comment by nicoaureliovillanueva** [comment]
@isnifer Can make a solution on this issue?

**Comment by isnifer** [comment]
@nicoaureliovillanueva you can read Twitter REST API docs and do it yourself.
